cmd/av/stack_diff.go

```diff
@@ -23,40 +25,25 @@ Generates the diff between the working tree and the current index (i.e., the dif
 			return err
 		}
 
-		diff, err := repo.Diff(&git.DiffOpts{})
+		currentBranchName, err := repo.CurrentBranchName()
 		if err != nil {
 			return err
 		}
 
-		diffLines := strings.Split(diff.Contents, "\n")
-
-		for _, line := range diffLines {
-			if strings.HasPrefix(line, "diff ") ||
-				strings.HasPrefix(line, "index ") ||
-				strings.HasPrefix(line, "+++") ||
-				strings.HasPrefix(line, "---") {
-				_, _ = fmt.Fprint(os.Stderr,
-					colors.Bold(line),
-					"\n",
-				)
-			} else if strings.HasPrefix(line, "+") {
-				_, _ = fmt.Fprint(os.Stderr,
-					colors.Success(line),
-					"\n",
-				)
-			} else if strings.HasPrefix(line, "-") {
-				_, _ = fmt.Fprint(os.Stderr,
-					colors.Failure(line),
-					"\n",
-				)
-			} else {
-				_, _ = fmt.Fprint(os.Stderr,
-					colors.Faint(line),
-					"\n",
-				)
-			}
+		branch, _ := meta.ReadBranch(repo, currentBranchName)
+
+		var opts = &git.DiffOpts{
+			Color: !color.NoColor,
+			Commit: branch.Parent.Name,
+		}
+
+		diff, err := repo.Diff(opts)
```

Comment:
nit: don't create a variable for `opts`, just declare the struct directly here. It's basically just the Go way of specifying keyword arguments.